AdyenComponents/BACS Direct Debit/BACSDirectDebitComponent.swift

```diff
@@ -50,23 +59,29 @@ public final class BACSDirectDebitComponent: PaymentComponent, PresentableCompon
     public init(paymentMethod: BACSDirectDebitPaymentMethod,
                 apiContext: APIContext,
                 style: FormComponentStyle = .init(),
-                localizationParameters: LocalizationParameters? = nil) {
+                localizationParameters: LocalizationParameters? = nil,
+                configuration: Configuration? = nil) {
```

Comment:
Optional `Configuration` means optional payment. Can this payment method work without a payment?

In other words: this is bank transfer; can you transfer no amount ?